docs/Installation.md

```diff
@@ -4,9 +4,8 @@
   - [Setting up the Databus](#setting-up-the-databus)
   - [Setting up the OPC UA Connector](#setting-up-the-opc-ua-connector)
   - [Setting up the Connection to the cloud](#setting-up-the-connection-to-the-cloud)
-    - [Creating a new policy for the aws IoT device](#creating-a-new-policy-for-the-aws-iot-device)
-    - [Creating a aws IoT Thing](#creating-a-aws-iot-thing)
-    - [Setting up the IE Cloud Connector](#setting-up-the-ie-cloud-connector)
+    - [Creating a new policy and thing for the aws IoT device](#creating-a-new-policy-and-thing-for-the-aws-iot-device)
+    - [Setting up the Cloud Connector](#setting-up-the-cloud-connector)
     - [Testing the Connection to the cloud](#testing-the-connection-to-the-cloud)
 
```

Comment:
IE Databus => Databus

Reply:
For example: `ie/d/j/simatic/v1/opcuac1/dp/r/plc1/tankLevel`
Is this topic correct?

Should it not be like this? Ending with default
`ie/d/j/simatic/v1/opcuac1/dp/r/plc1/default`

Reply:
IE Cloud Connector => Cloud Connector

Reply:
Enter the Databus User and Password in the Settings. "Bulk Publish" must be deactivated.

Thats not possible anymore.